### ISSUE TITLE
Add Erlang/Elixir client

### ISSUE THREAD

**Issue by mynameisdaniil** [MailRuChamps/raic-2020#72]
Hi! Would you consider adding Eralng/Elixir client if I create PR for it?
EDIT: Привет. Как насчет клиента для Эликсира/Эрланга? Добавите если сделаю? Есть описание протокола где-то или только реверсить код?

**Comment by kuviman** [comment]
Конечно. Если напишете в телеграм, можем сделать с автогенерацией, чтобы был доступен сразу в следующий раз.
